test/test_meta.py

```diff
@@ -1777,6 +1777,12 @@ def test_triangular_solve_out(self):
         self.assertEqual(out[1].shape, meta_out[1].shape)
         self.assertEqual(out[1].dtype, meta_out[1].dtype)
 
+    @onlyCPU
+    def test_linalg_eig_meta_stride_consistency(self):
+        x = torch.randn(5, 5, device='meta')
+        eigenvals, eigenvecs = torch.linalg.eig(x)
+        self.assertEqual(eigenvecs.stride(), (1, 5))
+
```

Comment:
You can add a test like https://github.com/pytorch/pytorch/pull/160894/files#diff-809c39aeafb3acc92289f42a63e670a8719d4ce5627d5f88820142d80edf8d2aR5720-R5731 that checks the strides for all device types.